modules/aws/aws_shared.go

```diff
@@ -0,0 +1,5 @@
+package aws
```

Comment:
was this meant to be included in the PR? (also the empty file.txt above)

Reply:
Yeah it was, ultimately we will want to share code and I was learning how to go about it in Go, it's empty for now but will need to be expanded upon later. I wanted to avoid putting my self in the situation of writing some tests and then having to fix what was wrong with it earlier.